ActivityLauncherApp/src/main/res/values/themes.xml

```diff
@@ -1,6 +1,14 @@
-<?xml version="1.0" encoding="utf-8"?>
-<resources>
-    <style name="AppTheme" parent="Theme.AppCompat.DayNight">
-        <item name="dialogCornerRadius">24dp</item>
-    </style>
-</resources>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This change seems unrelated to the shortcut creation issue. It's it necessary? Maybe put in separate commit/PR?

Reply:
Yes. When launching the activity, I could have extended FragmentActivity like it was for the root launcher and leave like that. The issue is that, when clicking on the shortcut, you can see for a brief moment, the actual Activity Launcher app. So instead we can extended AppCompatActivity with a custom theme so it will start smoothly